Assets/Scripts/PlayerUIManager.cs

```diff
@@ -23,26 +23,26 @@ public class PlayerUIManager : MonoBehaviour
 	    void Start()
 	    {
 	    	// Enable the Canvas and assign the main camera to it
-			Camera mainCamera = GameObject.FindGameObjectWithTag("MainCamera").GetComponent<Camera>();
+			Camera mainCamera = GameObject.FindGameObjectWithTag(Tags.MAIN_CAMERA).GetComponent<Camera>();
 			var canvas = GetComponent<Canvas>();
 			canvas.enabled = true;
 			canvas.worldCamera = mainCamera;
-            canvas.sortingLayerName = "Player";
+            canvas.sortingLayerName = Layers.PLAYER;
```

Comment:
sortinglayers aren't semantically the same thing as physics layers so it might be confusing to use a layers constant for sorting layers